modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java

```diff
@@ -691,7 +691,7 @@ private Map<String, String> _addClientExtensionEntries(
 					StringBundler.concat(
 						"[$CLIENT_EXTENSION_ENTRY_ID:",
 						jsonObject.getString("clientExtensionEntryKey"), "$]"),
-					jsonObject.getString("externalReferenceCode")));
+					String.valueOf(serviceContext.getCompanyId())));
```

Comment:
Shouldn't this be a concat of `ERC + _ + companyId` as in the rest of places? :thinking: 

Other than that it looks good to me :+1:

Reply:
Thanks!